<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ALF::nes_header_editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>65</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Filename:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Full path:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Prg size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Chr size:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_prg_size">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>300</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_filename">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>300</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_fullpath">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>300</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_chr_size">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>300</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mapper:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>90</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>SubMapper:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="battery">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Battery</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="trainer">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Trainer</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="mirroring">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Mirroring</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ignore_mirroring">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ignore mirror</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="mapper">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>100</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4095</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="submapper">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>190</y>
      <width>100</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>15</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="vs">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Vs Unisystem</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pc10">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PlayChoice-10</string>
    </property>
   </widget>
   <widget class="QComboBox" name="tv_system">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string notr="true">NTSC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">PAL</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TV System:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="dual">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Dual compatible</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="prgRamPresent">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Is PRG RAM present</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="busConflicts">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Bus conflicts?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>RAM battery backed:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="batteryBackedAmount">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>181</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1048576</number>
    </property>
    <property name="singleStep">
     <number>128</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RAM not battery backed:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="notBatteryBackedAmount">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>390</y>
      <width>181</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1048576</number>
    </property>
    <property name="singleStep">
     <number>128</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CHR not battery backed:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="chrBatteryBackedAmount">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>420</y>
      <width>181</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1048576</number>
    </property>
    <property name="singleStep">
     <number>128</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="chrNotBatteryBackedAmount">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>181</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1048576</number>
    </property>
    <property name="singleStep">
     <number>128</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CHR battery backed:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Vs Unisystem PPU:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="vsPPU">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>490</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0 - RP2C03B</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 - RP2C03G</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 - RP2C04-0001</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 - RP2C04-0002</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4 - RP2C04-0003</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 - RP2C04-0004</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6 - RC2C03B</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7 - RC2C03C</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8 - RC2C05-01</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9 - RC2C05-02</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10 - RC2C05-03</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11 - RC2C05-04</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12 - RC2C05-05</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13 - not defined</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14 - not defined</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15 - not defined</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Vs Unisystem mode:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="vsMode">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>520</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0 - Normal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 - RBI Baseball</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 - TKO Boxing</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 - Super Xevious</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4 - Not defined</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 - Not defined</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6 - Not defined</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7 - Not defined</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8 - Not defined</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9 - Not defined</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10 - Not defined</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11 - Not defined</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12 - Not defined</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13 - Not defined</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14 - Not defined</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15 - Not defined</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Misc. ROMs:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="miscRoms">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>560</y>
      <width>181</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
